jvb/src/main/kotlin/org/jitsi/videobridge/load_management/JvbLoadMeasurement.kt

```diff
@@ -24,6 +24,9 @@ interface JvbLoadMeasurement {
     fun getLoad(): Double
 
     operator fun div(other: JvbLoadMeasurement): Double
+    operator fun plus(other: JvbLoadMeasurement): Double {
```

Comment:
From an operator overloading point of view it's weird to have `plus` not return the type being added.